tsconfig-build.json

```diff
@@ -12,5 +12,6 @@
     {"path": "./packages/wonder-stuff-sentry"},
     {"path": "./packages/wonder-stuff-server"},
     {"path": "./packages/wonder-stuff-testing"},
+    {"path": "./packages/wonder-stuff-ci"},
```

Comment:
I was curious how the tsconfig-build.json is used in wonderstuff?

What else is needed for me to get this new package published to npm?

Reply:
All of it is managed by Changesets - once you changes are approved and landed then a Changesets PR will be generated, once that's approved and landed it'll publish to NPM!